src/home/frontend/components/notification/NotificationBell.js

```diff
@@ -130,7 +101,7 @@ const NotificationBell = ({ iconColor }) => {
     //Here navigate to the required notification
     convertStatus();
     listNotifbyUnseenStatus();
-    listNotifications123();
+    // listNotifications123();
```

Comment:
never use commented out code. comments should be used only for commenting purpose.